src/parser/jobs/nin/Weaving.js

```diff
@@ -84,6 +84,6 @@ export default class Weaving extends CoreWeaving {
 			return weaveCount > 1
 		}
 
-		return super.isBadWeave(weave, maxWeaves)
+		return super.isBadWeave(weave, 1)
```

Comment:
This doesn't concern me at all, that's what it's there for. I'll rely on you to know what is(n't) appropriate for NIN itself.